src/prajakta/Test1_NonRepeatingCharacter.java

```diff
@@ -0,0 +1,32 @@
+package prajakta;
+
+public class Test1_NonRepeatingCharacter{
+	 
+	int stringOccurance(String input, char ch) {
+		      int count = 0;
+		      for(int index =0; index<input.length(); index++) {
+		    	     if(input.charAt(index) == ch)
+		    	    	    count++;
+		      }
+		          return count;
+	}
+	
+	void firstSingleOccurance(String input) {
+		     char c = ' ';
+		     for (int index = 0; index< input.length(); index++) {
+		    	      char ch = input.charAt(index);
+		    	      int count = stringOccurance(input,ch);
```

Comment:
here you can use 
indexOf(ch) ==lastIndexOf(ch) logic to avoid one more for loop